src/codec.ts

```diff
@@ -52,6 +52,7 @@ export class NoiseHandshakeEncoder implements IEncoder {
       rateLimitProof: undefined,
       payload: this.hsStepResult.payload2.serialize(),
       version: version,
+      meta: undefined,
```

Comment:
We should open an issue to track the work with `meta` as per https://github.com/waku-org/js-waku/pull/1238

Reply:
We have [this one](https://github.com/waku-org/js-waku/issues/1208) or you want a separate to cover only `js-noise`?